### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by JeffersonBledsoe**
This PR fixes a few typos which existed in the current documentation. I've also added a header to the top-level README to keep [Markdown lint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041---first-line-in-a-file-should-be-a-top-level-heading) happy.

### CODE REVIEW

File: docs/themes/example-flixnet-ch4.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Component {
 }
 ```
 
-So which image asset should we use? A game box is a rectangle with 16:9 aspect ratio, so the `banner` would be perfect for this. However, since every asset is potentially missing, we should consider showing other images and provide multiple fallbacks. If we don't have a `banner`, the next similarly sized one is the `steam` ("grid icon") asset. Because it's wider than 16:9, we'll need to crop it if we don't want black bars or squashed/scretched images (though you might prefer that). If neither image is available, I'll use `boxFront` as it tends to be commonly available.
```
Comment (mmatyas) [inline docs/themes/example-flixnet-ch4.md:36]: Is the `fallbacks` change necessary?
There's also a space before `stretched`.

**Review by mmatyas — COMMENTED** [review]
Thank you! (I'm a bit embarrassed)
I've found just one section I have questions about.

File: docs/themes/example-flixnet-ch4.md (hunk 1 above)
Comment (JeffersonBledsoe) [reply docs/themes/example-flixnet-ch4.md:36]: Fixed, it was a force-of-habit for the space after the slash!
I've changed the fallbacks back to what it was previously too :)

Comment (mmatyas) [reply docs/themes/example-flixnet-ch4.md:36]: Thanks!

**Pull request merged**